weblab/entities/migrations/0009_remove_entity_visibility.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.13 on 2018-09-06 11:50
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('entities', '0008_auto_20180510_0824'),
+    ]
+
+    operations = [
```

Comment:
This looks like a likely culprit - no reverse migration!

Reply:
It shouldn't have one as it's an auto generated schema migration - Django will know to add the field back in when reversing.